libs/crypto/include/crypto/mcl_dkg.hpp

```diff
@@ -152,4 +153,96 @@ std::vector<DkgKeyInformation> TrustedDealerGenerateKeys(uint32_t committee_size
 
 }  // namespace mcl
 }  // namespace crypto
+
+namespace serializers {
+template <typename D>
+struct ArraySerializer<crypto::mcl::Signature, D>
```

Comment:
Side note: Signature here refers to the group signature right? Is it worth renaming this type to GroupSignature?

Reply:
It is for signature shares and the group signature as they are both of the same type.